tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateLikelySubtags.java

```diff
@@ -392,6 +393,9 @@ public static void main(String[] args) throws IOException {
                                 {"mro", "mro_Mroo_BD"},
                                 {"mro_BD", "mro_Mroo_BD"},
                                 {"ms_Arab", "ms_Arab_MY"},
+                                {"nan", "nan_Hans_CN"},
+                                {"nan_Hans", "nan_Hans_CN"},
```

Comment:
This won't hurt anything, but nan_Hans is redundant, because the algorithm will find {"nan", "nan_Hans_CN"}, and fill in.

There is a ticket open for dropping overrides that have no effect, so it is ok to keep this line for now.

Reply:
Interestingly, I need to keep this like otherwise in the produced `likelySubtags.xml` file, it will show `nan_Hans -> nan_Hans_TW` even though as we know the `Hant` script would be preferred in Taiwan. The problem is that we don't have population estimates on Simplified v Traditional Chinese script usage.